telemetry/shield-study-addon.schema.json

```diff
@@ -1,81 +1,266 @@
+                            "type": "string"
+                        },
+                        "defaultSearchEngineData": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string"
+                                },
+                                "loadPath": {
+                                    "type": "string"
+                                },
+                                "submissionURL": {
+                                    "type": "string"
+                                },
+                                "origin": {
+                                    "type": "string"
+                                }
+                            }
+                        },
+                        "e10sEnabled": {
+                            "type": "boolean"
+                        },
+                        "e10sCohort": {
+                            "type": "string"
+                        },
+                        "locale": {
+                            "type": "string"
+                        },
+                        "telemetryEnabled": {
+                            "type": "boolean"
+                        },
+                        "update": {
+                            "type": "object",
+                            "properties": {
+                                "autoDownload": {
+                                    "type": "boolean"
+                                },
+                                "channel": {
+                                    "type": "string",
+                                    "enum": [
+                                        "nightly",
+                                        "aurora",
+                                        "beta",
+                                        "release"
+                                    ]
+                                },
+                                "enabled": {
+                                    "type": "boolean"
+                                }
+                            }
+                        }
                     }
                 }
             },
             "required": [
-                "attributes"
+                "system",
+                "profile",
+                "settings"
             ]
         },
+        "id": {
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
+        },
         "type": {
             "type": "string",
             "description": "doc_type, restated",
             "enum": [
                 "shield-study-addon"
             ]
+        },
+        "version": {
+            "type": "number",
+            "minimum": 4,
+            "maximum": 4
+        },
+        "payload": {
+            "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
This `$schema` field needs to go back to the top-level of the schema [here](https://github.com/mozilla-services/mozilla-pipeline-schemas/pull/26/files#diff-cf02c0cb76e53d1738f859bba28b6f02L2)